config/application.rb

```diff
@@ -34,10 +34,15 @@ def self.included(base)
   mattr_accessor :redis
 
   class Application < Rails::Application
+    def config_for(*args)
+      config = super
+      return config unless config.is_a?(Hash)
+      config.with_indifferent_access
+    end
 
     def simple_try_config_for(*args)
       config_for(*args)
-    rescue ConfigFor::ReadError, ConfigFor::MissingEnvironmentError => error
+    rescue => error
```

Comment:
Does this mean the same as `rescue Exception => error` ? In that case:
1. I think it is better to specify it this way to it is clearer, but this is not the biggest issue about this.
2. @hallelujah Told me once that the problem of catching everything is that is literally eveything, including doing a `CTRL + C`, so I am not sure this is a very good idea :thinking:

Reply:
Not really you cannot rescue it see how Rails config_for is done.